var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -64,6 +64,12 @@ class Libxsmm(MakefilePackage):
             description='With header-only installation')
     variant('generator', default=False,
             description='With generator executable(s)')
+    variant('BLAS', default='default', multi=False,
+            description='Control behavior of BLAS calls',
+            values=('default', '0', '1', '2'))
+    variant('CODE_BUF_MAXSIZE', default=0, multi=False,
```

Comment:
We try to keep variants lowercase

Reply:
I agree in general, it was not my first choice. Unfortunately these two options that are exposed here are a bit esoteric, and I think that the capitlization associates these options directly with their role in the installation instructions of the application, as well as in the Makefile. 
I feel that in this situation, lowercase will be more confusing for users (fe.g +blas here does not mean the same as other applications).
@hfp @alheinecke let me know what you would like us to do here. Thanks.